src/utils/sorting-functions.ts

```diff
@@ -0,0 +1,123 @@
+import { ToDoItem, Priority } from '../models/todoitem-model';
+
+export function sortByTitle(items: ToDoItem[], order: number): ToDoItem[] {
+  const orderedItems = items;
+  if (order === 1) {
+    orderedItems.sort((a, b) => {
+      const titleA = a.title.toLowerCase();
+      const titleB = b.title.toLowerCase();
+
+      if (titleA < titleB) {
+        return -1;
+      }
+      if (titleA > titleB) {
+        return 1;
+      }
+      return 0;
+    });
+  } else {
+    orderedItems.sort((a, b) => {
+      const titleA = a.title.toLowerCase();
+      const titleB = b.title.toLowerCase();
+
+      if (titleA > titleB) {
+        return -1;
+      }
+      if (titleA < titleB) {
+        return 1;
+      }
+      return 0;
+    });
+  }
+  return orderedItems;
+}
+
+export function sortByText(items: ToDoItem[], order: number): ToDoItem[] {
+  const orderedItems = items;
+  if (order === 1) {
```

Comment:
Same here:  the sort functionality is almost the same. Please refactor this file, create a helper function for sorting and use it instead of writing almost the same code block multiple times.